src/main/java/org/wahlzeit_revisited/service/Transformer.java

```diff
@@ -0,0 +1,63 @@
+package org.wahlzeit_revisited.service;
+
+import jakarta.inject.Inject;
+import jakarta.inject.Singleton;
+import org.wahlzeit_revisited.dto.PhotoDto;
+import org.wahlzeit_revisited.dto.UserDto;
+import org.wahlzeit_revisited.model.Photo;
+import org.wahlzeit_revisited.model.User;
+import org.wahlzeit_revisited.utils.WahlzeitConfig;
+
+import java.io.File;
+
+@Singleton
+public class Transformer {
+
+    @Inject
+    public WahlzeitConfig config;
+
+    public UserDto transform(User user) {
+        assertNonNull(user);
```

Comment:
Why "assertNonNull" rather than "assertIsNotNull"? The implementation could/would use isNotNull rather a query method called "NonNull".

Reply:
changed in: efdc95e1d2b5126e4b2b4a1932d185ca6b6a59bd